src/core/function_shared.tf

```diff
@@ -0,0 +1,54 @@
+resource "azurerm_resource_group" "shared_rg" {
+  name     = format("%s-shared-rg", local.project)
+  location = var.location
+}
+
+resource "azurerm_app_service_plan" "shared_plan_1" {
+  name                = format("%s-plan-shared-common-1", local.project)
+  location            = azurerm_resource_group.shared_rg.location
+  resource_group_name = azurerm_resource_group.shared_rg.name
+
+  kind     = var.plan_shared_kind
+  reserved = true
+
+  sku {
+    tier     = var.plan_shared_sku_tier
+    size     = var.plan_shared_sku_size
+    capacity = var.plan_shared_sku_capacity
+  }
+
+  tags = var.tags
+}
+
+# Subnet to host app function
+module "shared_snet" {
+  source                                         = "git::https://github.com/pagopa/azurerm.git//subnet?ref=v1.0.51"
+  name                                           = format("%s-shared-snet", local.project)
```

Comment:
```suggestion
  name                                           = format("%s-shared-1-snet", local.project)
```

Reply:
solved in https://github.com/pagopa/io-infra/pull/316/commits/46bd3249e99fe4ac8b3562c8c766c5af51e804da